hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMessageTask.java

```diff
@@ -76,17 +76,17 @@ protected AbstractMessageTask(ClientMessage clientMessage, Node node, Connection
         this.clientMessage = clientMessage;
         this.logger = node.getLogger(getClass());
         this.node = node;
-        this.nodeEngine = node.nodeEngine;
```

Comment:
Are these changes related to the rest of the PR?

Reply:
Not much related, but I guess the person who did the fix fixed these along the way. I guess they are to use getters instead of using fields directly.